src/docs/src/whatsnew/3.3.rst

```diff
@@ -0,0 +1,263 @@
+.. use this file except in compliance with the License. You may obtain a copy of
+.. the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+.. _release/3.3.x:
+
+============
+3.3.x Branch
+============
+
+.. contents::
+    :depth: 1
+    :local:
+
+.. _release/3.3.0:
+
+Version 3.3.0
+=============
+
+Features and Enhancements
+-------------------------
+
+* :ghissue:`3746`: Remove couch_icu_driver module and re-use ``couch_ejson_compare`` functionality.
+
+* :ghissue:`3776`, :ghissue:`3970`, :ghissue:`3972`, :ghissue:`4093`, :ghissue:`4102`, :ghissue:`4104`, :ghissue:`4110`, :ghissue:`4111`, :ghissue:`4114`, :ghissue:`4245`, :ghissue:`4246`: Add ``smoosh`` queue persistence. This allows resuming ``smoosh`` operations after a node restart. This is disabled by default and can be enabled with ``[smoosh] persist = true``.
+
+* :ghissue:`3798`: Add ``libicu`` version and collation algorithm version to ``/_node/_local/_versions``.
+
+* :ghissue:`3837`: The Erlang source tree is now auto-formatted with ``erlfmt``.
+
+* :ghissue:`3845`: Clean up the ``couch_ejson_compare`` C-module and squash Microsoft compiler warnings.
+
+* :ghissue:`3832`: Add ``GET`` variant to ``_dbs_info`` endpoint, used to be ``POST`` only.
+
+* :ghissue:`3864`: Improve ``erlang_ls`` configuration.
+
+* :ghissue:`3853`: Remove legacy ``ddoc_cache_opener`` gen_server and speed up event routing.
+
+* :ghissue:`3879`: Remove use of ``ERL_OPTS`` environment variable. All supported Erlang version now use ``ERL_COMPILER_OPTIONS`` for the same purpose.
+
+* :ghissue:`3883`: Add support for SpiderMonkey 91.
+
+* :ghissue:`3889`: Track ``libicu`` collator versions in the view header
+
+* :ghissue:`3925`: Make the timeout for receiving requests from attachment writers configurable.
+
+* :ghissue:`3927`: Include index signature in ``_search_info``.
+
+* :ghissue:`3963`: Optimtize key tree stemming by using maps instead of sets. This greatly reduced memory usage for heavily conflicted docs in some situations.
+
+* :ghissue:`3974`: Create new config options in ``[couchdb]`` and ``[smoosh]`` sections to enable finer control of compaction logging levels.
+
+* :ghissue:`3983`, :ghissue:`3984`, :ghissue:`3985`, :ghissue:`3987` :ghissue:`#4033`: Add various functions to ``couch_debug`` module.
+
+* :ghissue:`4000`: Ensure ``Object.prototype.toSource()`` is always available
+
+* :ghissue:`4017`: Update ``mochiweb`` to 3.1.1.
+
+* :ghissue:`4018`: Update ``jiffy`` to 1.1.1 and ``b64url`` to 1.0.3.
+
+* :ghissue:`4021`: Reduce smoosh compaction log level to ``debug``.
+
+* :ghissue:`4135`: Allow and evaluate nested json claim roles in JWT token.
```

Comment:
#4135 is: add ioq:call_search

#4041 Allow and evaluate nested json claim roles